app.js

```diff
@@ -0,0 +1,15 @@
+const express = require("express");
+const { handle404, handleServerError } = require("./errors");
+const { getTopics } = require("./controllers/topics.controllers");
+
+const app = express();
+
+app.use(express.json());
```

Comment:
We aren't yet using a post request so this line should be removed
It's important to only add code that meets the need of the current task/ticket